CONTRIBUTING.md

```diff
@@ -0,0 +1,6 @@
+# Contributing
+
+Contributions to the Adfinis Helm Charts are very welcome! Have a look at the open [issues](https://github.com/adfinis-sygroup/helm-charts/issues)
+and open a [GitHub pull request](https://github.com/adfinis-sygroup/helm-charts/compare). See our [docs](./docs/development.md) for how to setup a
```

Comment:
we are in fact on GitHub, do we need to tell people to open a "GitHub pull request"? IMHO "open a pull request" would suffice.